### PR TITLE
[typo](docs) Fix some errors in the description

### PR DESCRIPTION
**Pull request by yagagagaga**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ under the License.
 | 1    | 100  | beijing   | 1.1  |
 | 2    | 200  | shanghai  | 1.2  |
 | 3    | 300  | guangzhou | 1.3  |
-| 4    | 400  | chongqing | 1.4  |
+| 4    | \N   | chongqing | 1.4  |
```
Comment (zy-kkk) [inline docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md:198]: In the result below, column k3 has 40,000, which is obviously the second column with the original data multiplied by 100. Why did you change it to null?
You should change the 4 in the first column to null

File: docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ under the License.
...
```
Comment (zy-kkk) [inline docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md:289]: Again, the filtered data is 400, which should not be changed to null.
You should change the 4 in the first column to null

**Review by zy-kkk — CHANGES_REQUESTED** [review]

File: docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md (hunk 1 above)
Comment (yagagagaga) [reply docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md:198]: Right,  I change it wrong

File: docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md (hunk 2 above)
Comment (yagagagaga) [reply docs/zh-CN/docs/data-operate/import/import-scenes/load-data-convert.md:289]: Yes, I found that I corrected the wrong place.
